.github/workflows/e2e-test-event-writer.yml

```diff
@@ -0,0 +1,87 @@
+name: Build E2E Test Event Writer
```

Comment:
Can you add a .md file describing the test architecture explaining various components involved and the expected outputs you are validating. It will be useful for to make future changes.